### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rbatistadev**
#### Type Of Change 
- [X] Small bug fix
- [ ] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, portal/platform, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Updated ivoz-dev-tools dependencies

### CODE REVIEW

**Review by ironArt3mis — CHANGES_REQUESTED** [review]
Functional review required

File: library/Ivoz/Provider/Domain/Model/Administrator/AdministratorSecurityTrait.php
Comment (danigargar) [inline library/Ivoz/Provider/Domain/Model/Administrator/AdministratorSecurityTrait.php:1]: This is not autogenerated. Use a separate commit, please

File: library/composer.lock
Comment (danigargar) [inline library/composer.lock:1]: Why there are no changes to composer.json?

**Review by danigargar — CHANGES_REQUESTED** [review]

File: library/composer.lock
Comment (rbatistadev) [reply library/composer.lock:1]: because there is a mask in composer.json, something like irontec/ivoz-dev-tools:^5.3

**Review by danigargar — APPROVED** [review]

**Review by danigargar — APPROVED** [review]

**Review by ironArt3mis — APPROVED** [review]

**Pull request merged**